Adding tuxmake environment names as qemu-*

nareshkamboju opened the pull request:
Following test environments added to known issues test list
  - qemu-x86_64
  - qemu-i386
  - qemu-armv7
  - qemu-arm64

Suggested-by: Anders Roxell <anders.roxell@linaro.org>

roxell commented:
1. no need for the link in the changelog, please remove.
2. "Adding tuxmake environment..." should be "Adding tuxtest environment...".
3. qemu-arm should be qemu-armv7 for now, maybe also qemu-armv5...

nareshkamboju commented:
@roxell Thanks for your comments.
I have addressed your review comments and pushed a new version of patch.
And We are doing only build testing on qemu-armv5. So we do not have to add qemu-armv5 as environment.
